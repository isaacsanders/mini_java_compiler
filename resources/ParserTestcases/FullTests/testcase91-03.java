/*this is a test*/
class Test {
	public static void main (String[] args) {
		System.out.println(2 + 13); // cool
	}
}

class Test ing{//broken!
	public static void main(String[] args) {
		int oops=0;
		int i=0;
		int shouldnt=0;
		int be=0;
		int here;//brokenmore!
	}
	

	public static ChallengeYouToExitGracefully((){
	{{{{{
		{{{{{
			{{{{{//oops I thought I commented those out
}
