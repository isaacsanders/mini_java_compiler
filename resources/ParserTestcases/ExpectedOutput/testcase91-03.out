StmtList ::= 
LiteralExpr ::= INTEGER
CallExpr ::= LiteralExpr
UnaryExpr ::= CallExpr
Expr ::= UnaryExpr
LiteralExpr ::= INTEGER
CallExpr ::= LiteralExpr
UnaryExpr ::= CallExpr
Expr ::= UnaryExpr
Expr ::= Expr PLUS Expr
Stmt ::= SYSTEMOUTPRINTLN LPAREN Expr RPAREN SEMI
StmtList ::= StmtList Stmt
MainClassDecl ::= ...
ClassDeclList ::= 


ERROR in line 8 and column 12: Syntax error



ERROR in line 8 and column 12: Couldn't repair and continue parse

Exception processing file: Can't recover from previous error(s)
